_content/annotated-translation-of-the-chinese-version-of-the-kayagatasati-sutta_kuan-tsefu

```diff
@@ -0,0 +1,19 @@
+---
+title: "Annotated Translation of the Chinese Version of the Kāyagatāsati Sutta"
+authors:
+  - kuan-tsefu
+external_url: "https://www.academia.edu/34449156/Annotated_Translation_of_the_Chinese_Version_of_the_K%C4%81yagat%C4%81sati_Sutta"
+drive_links:
+  - "https://drive.google.com/file/d/1RqMj8T6F0oVb_1I18EjOXtm5q5bf-KLT/view?usp=drivesdk"
+course: sati
+tags:
+  - sutta
+  - kaya
+  - jhana
+year: 2007
+journal: iijbs
+volume: 8
+pages: "175--194"
+---
+
+A translation of the Chinese Version of the Kāyagatāsati Sutta (MA 81), which differs considerably from the Pāli version of the text. This Chinese version of the sutta is found in the Madhyama Āgama which is conisdered to be influnced by the Sarvastivada school.
```

Comment:
No need for the "This Chinese..." sentence, and please link "the Pāli version of the text" to https://buddhistuniversity.net/content/canon/mn119 like so:

```markdown
A translation of the Chinese Version of the Kāyagatāsati Sutta (MA 81),
which differs considerably from [the Pāli version of the text](/content/canon/mn119).
```